website/api/zkvm/composition.md

```diff
@@ -0,0 +1,55 @@
+# Proof Composition
+
+RISC Zero's zkVM offers a convenient API for efficiently verifying proofs inside the guest program.
+
+This feature enables rich possibilities for building modular applications. Check out the [blog post] to learn more about the power of proof composition.
+
+:::info
+
+Although the "obvious" approach to proof composition would be to simply run the verifier inside the guest program, this approach is very expensive.
```

Comment:
It's actually not so expensive that there are not cases where folks will use this approach. In particular, there is https://github.com/risc0/risc0/issues/1355. The real core reason is you don't get compression because verifying two receipts takes more than two segments, and so generates more than two receipts. You need verification of two receipts to generate one receipt.
```suggestion
Although the "obvious" approach to proof composition would be to simply run the verifier inside the guest program, this approach doesn't result in compression.
```